src/react/components/pages/connections/connectionForm.tsx

```diff
@@ -2,11 +2,47 @@ import React from "react";
 import Form, { Widget } from "react-jsonschema-form";
 import { IConnection } from "../../../../models/applicationState.js";
 import LocalFolderPicker from "../../common/localFolderPicker";
-// tslint:disable-next-line:no-var-requires
-const formSchema = require("./connectionForm.json");
+import { strings } from "../../../../common/strings";
 // tslint:disable-next-line:no-var-requires
 const uiSchema = require("./connectionForm.ui.json");
 
+const formSchema = {
```

Comment:
I'd like to see if we can somehow references the localized strings from within the JSON in some sort of variables notation like `${strings.common.displayName}`

Reply:
Then we can run the object through some parser that does some deep scanning / replacements?